tests/openwrt/test_encryption.py

```diff
@@ -654,3 +916,248 @@ def test_render_wps_ap(self):
     def test_parse_wps_ap(self):
         o = OpenWrt(native=self._wps_ap_uci)
         self.assertEqual(o.config, self._wps_ap_netjson)
+
+    def test_render_ieee80211w(self):
+        _netjson_wpa3_personal_cipher_tkip_ieee80211_none = {
+            "interfaces": [
+                {
+                    "name": "wlan0",
+                    "type": "wireless",
+                    "wireless": {
+                        "radio": "radio0",
+                        "mode": "access_point",
+                        "ssid": "wpa3-personal",
+                        "encryption": {
+                            "protocol": "wpa3_personal",
+                            "cipher": "tkip",
```

Comment:
This is a wrong configuration. Is it done purposefully to show that the resulting configuration will use `ccmp` cipher?

Reply:
Yes, one of the purposes of this test is to make sure that TKIP can be replaced by CCMP.